nanoFramework.System.Net.Http/Http/System.Net.HttpListenerRequest.cs

```diff
@@ -121,8 +122,20 @@ public void Reset()
         /// </summary>
         internal void ParseHTTPRequest()
         {
-            // This is the request line.
-            m_RequestString = m_clientStream.Read_HTTP_Line(HttpConstants.maxHTTPLineLength).Trim();
+            try
+            {
+                // This is the request line.
+                m_RequestString = m_clientStream.Read_HTTP_Line(HttpConstants.maxHTTPLineLength).Trim();
+            }
+            catch (SocketException)
+            {
```

Comment:
Just need a comment here with a brief explanation on why we are choosing to catch this type of exception and why it's OK to ignore is.